src/App/Expression.cpp

```diff
@@ -2344,6 +2512,12 @@ Py::Object FunctionExpression::evaluate(const Expression *expr, int f, const std
     case FLOOR:
         output = floor(value);
         break;
+    case ROTATIONX:
+    case ROTATIONY:
+    case ROTATIONZ:
+        return Py::asObject(new Base::RotationPy(Base::Rotation(Vector3d(f == ROTATIONX, f == ROTATIONY, f == ROTATIONZ), value)));
```

Comment:
To me this would be clearer if you `static_cast<int>` these, otherwise it looks like you're trying to create a vector of booleans.

Reply:
Thanks, just added static casts for these values.